src/components/DestinationBox.jsx

```diff
@@ -0,0 +1,49 @@
+import React, {useState, useEffect} from "react";
+
+export default function DestinationBox({destination, onDelete}) {
```

Comment:
as you're using ES6 in all the rest of the code, you can apply it to functional components as well and write
`export const DestinationBox = ({ destination, onDelete }) => { ...`
just to be consistent